PLUB/Sources/Views/Home/ViewModel/SelectedCategoryViewModel.swift

```diff
@@ -6,25 +6,67 @@
 //
 
 import RxSwift
+import RxCocoa
 
 protocol SelectedCategoryViewModelType {
+  // Input
+  var selectCategoryId: AnyObserver<String> { get }
+  
+  // Output
+  var updatedCellData: Driver<[SelectedCategoryCollectionViewCellModel]> { get }
+  
   func createSelectedCategoryChartCollectionViewCellModels() -> Observable<[SelectedCategoryCollectionViewCellModel]>
 }
 
 class SelectedCategoryViewModel: SelectedCategoryViewModelType {
   
+  private var disposeBag = DisposeBag()
+  
+  // Input
+  let selectCategoryId: AnyObserver<String>
+  
+  // Output
+  let updatedCellData: Driver<[SelectedCategoryCollectionViewCellModel]>
+  
+  init() {
+    let selectingCategoryId = PublishSubject<String>()
+    let updatingCellData = BehaviorSubject<[SelectedCategoryCollectionViewCellModel]>(value: [])
+    self.selectCategoryId = selectingCategoryId.asObserver()
+    self.updatedCellData = updatingCellData.asDriver(onErrorDriveWith: .empty())
+    
+    let fetchingSelectedCategory = selectingCategoryId.flatMapLatest { categoryId in
+      return MeetingService.shared.inquireCategoryMeeting(categoryId: categoryId)
+    }
+    .share()
+
+    let successFetching = fetchingSelectedCategory.map { result -> CategoryMeetingResponse? in
+      guard case .success(let response) = result else { return nil }
+      return response.data
+    }
+    
+    let selectingContents = successFetching.map { response -> [Content]? in
+      guard let response = response else { return nil }
+      return response.content
+    }
```

Comment:
이건 compactMap으로 줄일 수 있지 않나요?
```suggestion
    let selectingContents = successFetching.compactMap { $0?.content }
```
이렇게요!

Reply:
엇 넵 맞습니다 !! compactMap으로  하면 좀 더 깔끔하게 할 수 있겠네요 !!